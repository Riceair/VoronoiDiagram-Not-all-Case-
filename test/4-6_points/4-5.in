4
243 266
136 363
210 240
321 396