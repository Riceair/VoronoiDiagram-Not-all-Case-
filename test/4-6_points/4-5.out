P 136 363
P 210 240
P 243 266
P 321 396
E -77317 -46318.5 183.35 307.73
E -20561.5 116929.5 231.77 361.14
E 183.35 307.73 231.77 361.14
E 183.35 307.73 16606.5 -20537
E 231.77 361.14 82182 -48809
